django/api/tasks.py

```diff
@@ -362,3 +366,36 @@ def check_rebates_redeemed_since(iso_ts=None, schedule_func_name=None):
         status=GoElectricRebateApplication.Status.REDEEMED,
         modified=timezone.now(),
     )
+
+
+# Auto-Cancel "household" applications in "initiated" status for more than 28 days
+def cancel_household_applications_initiated_status():
+    # get all applications in "initiated" status
+    applications = GoElectricRebateApplication.objects.filter(
+        status=GoElectricRebateApplication.Status.HOUSEHOLD_INITIATED
+    )
+
+    # get all applications that are "household" and "initiated" for more than 28 days
+    household_applications_initiated_28_days = applications.filter(
+        modified__lte=timezone.now() - timedelta(days=28)
```

Comment:
Shouldn't we use the "created" field here, unless you have a specific reason for using "modified"?